std/node/_fs/_fs_writeFile_test.ts

```diff
@@ -101,7 +101,7 @@ Deno.test(
       read: true,
     });
 
-    await new Promise((resolve, reject) => {
+    await new Promise<void>((resolve, reject) => {
```

Comment:
Is type parameter now mandatory for `Promise`?

Reply:
If it cannot be inferred, yes. https://devblogs.microsoft.com/typescript/announcing-typescript-4-1/#resolves-parameters-are-no-longer-optional-in-promises. A good type safety thing that avoids a lot of errors of resolving promises incorrectly.